oleesir/authors-haven
# Logout user

**What does this Merge request do?**

Implements ffeat__logoutUser

**Description of Tasks completed?**

Implements ffeat__logoutUser

**How should this be manually tested?**

- git fetch on your local machine and check out to feat__logoutUser branch
- Run "yarn start" to start the development server

**Any background context you want to provide?**

N/A


**Screenshots (if appropriate)**
N/A
